Makefile-libostree.am

```diff
@@ -260,6 +260,20 @@ libostree_1_la_CFLAGS += $(OT_DEP_SELINUX_CFLAGS)
 libostree_1_la_LIBADD += $(OT_DEP_SELINUX_LIBS)
 endif
 
+libostree_1_la_SOURCES += \
+	src/libostree/ostree-sign.c \
+	src/libostree/ostree-sign.h \
+	src/libostree/ostree-sign-dummy.c \
+	src/libostree/ostree-sign-dummy.h \
+	src/libostree/ostree-sign-ed25519.c \
```

Comment:
Maybe this (and the .h) should be wrapped by `USE_LIBSODIUM`

Reply:
No, they must exists. Functions from `*ed25519.c` ruturn error "Have no libsodium support" if build without it.